### ISSUE TITLE
[3.8] Potential session problem

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#16477]
This could be a false positive, this could be a legit issue, but on my local clone of the downloads site where I am running the 3.8 dev build, I am getting this message somewhat consistently when I first open the app.

```sh
Warning: session_start(): The session id is too long or contains illegal characters, valid characters are a-z, A-Z, 0-9 and '-,' in /libraries/joomla/session/handler/native.php on line 260
```

**Comment by andrepereiradasilva** [comment]
i had this issue too in **latest staging** using php sessions, not sure how it appeared.

**Comment by andrepereiradasilva** [comment]
@mbabker i can reproduce an issue in **latest staging**

1. Set the session handler to PHP (files handler)
2. Set session timeout to 1 minute (for faster test)
3. Login in backend
4. Wait around a minute
5. Refresh the page

>Warning: session_start(): The session id is too long or contains illegal characters, valid characters are a-z, A-Z, 0-9 and '-,' in /path/to/joomla/staging/libraries/joomla/session/handler/native.php on line 260 
> 
> Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /path/to/joomla/staging/libraries/joomla/session/handler/native.php:260) in /path/to/joomla/staging/libraries/joomla/session/handler/native.php on line 260
> 
> Warning: session_regenerate_id(): Cannot regenerate session id - headers already sent in /path/to/joomla/staging/libraries/joomla/session/handler/native.php on line 151
> 
> Warning: session_write_close(): The session id is too long or contains illegal characters, valid characters are a-z, A-Z, 0-9 and '-,' in /path/to/joomla/staging/libraries/joomla/session/handler/native.php on line 154
> 
> Warning: session_write_close(): Failed to write session data (files). Please verify that the current setting of session.save_path is correct (/path/to/joomla/staging/tmp) in /path/to/joomla/staging/libraries/joomla/session/handler/native.php on line 154
> 
> Error displaying the error page: Application Instantiation Error: Failed to start the session because headers have already been sent by "/path/to/joomla/staging/libraries/joomla/session/handler/native.php" at line 260.

Could this be related to the cookies PR https://github.com/joomla/joomla-cms/pull/12306 ?

**Comment by andrepereiradasilva** [comment]
@mbabker please test https://github.com/joomla/joomla-cms/pull/16481

**Comment by franz-wohlkoenig** [comment]
closed as having PR #16481<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16477">issues.joomla.org/tracker/joomla-cms/16477</a>.</sub>
